core/eolearn/core/utilities.py

```diff
@@ -7,6 +7,7 @@
 from collections import OrderedDict
 
 import numpy as np
+import cv2
```

Comment:
Unfortunately OpenCV is not a dependency of `eo-learn-core` and I'm not sure if it should be.

Suggestion:
- This utility is for now moved to `eo-learn-mask`. In it will be needed by other subpackages in the future we can move it to core.
- Dependency `opencv-contrib-python-headless` is added to `eo-learn-mask`